test/model-helpers/Profile.test.js

```diff
@@ -0,0 +1,30 @@
+import { beforeEach, afterEach, describe, expect, vi, it } from 'vitest';
+import { fetchProfileInfo } from '../../src/model-helpers';
+
+let session = {};
+
+vi.mock('@inrupt/solid-client');
+
+describe('fetchProfileInfo', () => {
+  beforeEach(() => {
+    session = {
+      fetch: vi.fn(),
+      info: {
+        webId: 'https://newtestuser2.opencommons.net/profile/card#me'
```

Comment:
Would be better not to use a real webId in tests. That way no one will get it confused with real data.

Reply:
Think I'll just mock the ttl file. I'll give it a go.